assignment5.txt

```diff
@@ -0,0 +1,11 @@
+Drew French
+
+1)	Passing by value: Passing a value to something.
```

Comment:
Give a more thorough explanation here - pass by value works by passing a copy of the original value.